.github/workflows/release.yaml

```diff
@@ -0,0 +1,63 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  Release:
+    name: 'Release'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: 'marvinpinto/action-automatic-releases@latest'
+        with:
+          repo_token: '${{ secrets.GITHUB_TOKEN }}'
+          prerelease: false
+
+  Docker-Build-Deploy:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
Do we need this?  This whole workflow should only run on tags already.

Reply:
I don't think we need it, I kept it because I don't see the benefit of starting this if tags are pushed to anything but main.

Reply:
Add the `main` branch as an additional requirement at the top where you do the `v**` bit